ddl/ddl_api.go

```diff
@@ -462,14 +475,13 @@ func checkDefaultValue(ctx context.Context, c *table.Column, hasDefaultValue boo
 	return nil
 }
 
-func checkDuplicateColumn(colDefs []*ast.ColumnDef) error {
+func checkDuplicateColumn(cols []string) error {
 	colNames := map[string]bool{}
-	for _, colDef := range colDefs {
-		nameLower := colDef.Name.Name.L
-		if colNames[nameLower] {
-			return infoschema.ErrColumnExists.GenByArgs(colDef.Name.Name)
+	for _, col := range cols {
+		if colNames[strings.ToLower(col)] {
+			return infoschema.ErrColumnExists.GenByArgs(model.NewCIStr(col))
 		}
-		colNames[nameLower] = true
+		colNames[strings.ToLower(col)] = true
```

Comment:
We can make sure the input is lower and avoid `ToLower`?

Reply:
yes，we can do it outside to make sure the input is lower, but i think anyone need to know the input is lower and convert  it outside if we do not use _ToLower_

Reply:
We use `col.Name.Name.L`. So can we remove `ToLower`?

Reply:
Use col.Name.Name.L is better.